tokens/movistar.json

```diff
@@ -1126,6 +1126,24 @@
         "value": "bold",
         "type": "typography"
       }
+    },
+    "size": {
+      "tabsLabel": {
+        "value": {
+          "mobile": 16,
```

Comment:
should we change the type of value here to `"16"` to use "string" type in patterns?

Reply:
I've change to use enum instead of pattern and it works in the same way, so we can decide either if we want to use number or string type.

Reply:
When talking about unit-less values for dimensions, it just feels more correct to use a number instead of a string. I have my doubts looking to the border radii definition, since they can be potentially numbers too...but then we have "circle"

Reply:
border radius also support values like "16 16 0 16"

An alternative is use the following structure for border radius patternProperties

```
"value": {
          "type": ["string", "number"],
          "enum": [0, 2, 4, 8, 12, 16, 999, "circle"]
        },
```

Reply:
> lineHeight in this case will be number type, right?

Yes, that was just and example